DeAceroDLabs/marketplace-frontend
# [DMP-465] Added external number field

## What is done

[[DMP-465] Agregar Unit Tests al Signup UI](https://deacero.atlassian.net/browse/DMP-465)

## How is done

1. Added a new field as a TextField in the "ubicación" section on the sign-up.
2. Added new prop `size` to the fields, so it can reduce the size of the field if necessary.
3. Set the size of the street field as "medium" and the external number field as small so they could fit in one row.

## How to test

1. run `npm start`.
2. go to `/signup`.
3. check the new field and layout of the "ubicación" section of the form.
